taozhaolib/islandora_internet_archive_bookreader_custom
# Islandora bookreader PDF generator and downlink block update

1. Changed the strategy to configure the display of the custom features;
2. Changed the strategy to display the download page links;
3. Added the access control to PDF generator and download link in the block;
4. Some other small fixes and updates.

How to test:
1. Download link block and PDF generator should only appear for book object pages;
2. Download link block and PDF generator should appear according to the configuration settings;
    2.1 When PDF generator and PDF download link both disappear when PDF download link is set to be not appearing;
3. PDF generator should work properly and is redirected to book view page after done;
4. PDF download link and page download links should work properly when clicked;
5. When change book page display, the book page download link should change accordingly.
